source/beamline/AMDetector.cpp

```diff
@@ -428,28 +459,34 @@ bool AMDetector::clear(){
 	return clearImplementation();
 }
 
-void AMDetector::setAsDarkCurrentMeasurementValue(){
+void AMDetector::setLastMeasurementAsDarkCurrent()
+{
+	if (canDoDarkCurrentCorrection()) {
+		double newTime = acquisitionTime();
+		double newValue = double(singleReading()) / newTime;
+
+		updateDarkCurrentTime(newTime);
+		updateDarkCurrentValue(newValue);
+		updateDarkCurrentValidState(true);
```

Comment:
Ah, not really.  I didn't realize that one was public whereas the other was protected.  It should be fine this way.